content/reference/user/profile/index.md

```diff
@@ -9,17 +9,20 @@ weight: 10030
 
 ## Accessing your profile
 
-You can view your profile at any time by clicking on the **My Profile** button in the navigation bar.
+{{< img src="profile_menu.png" alt="My profile button" >}}
 
-{{< img src="profile.png" alt="Profile" >}}
+You can view your profile at any time by clicking on the **My profile** button, from the drop-down menu available by clicking
```

Comment:
My profile is not on top right dropdown (organization now) but in the left bar

Reply:
It is in the top right menu. I even added a screenshot to show where it is.

Reply:
My bad (and I was on phone)
I was mislead by the Users icon selected in the left bar